Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/) for Kodi add-ons: skins/skin.estuary.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-skins/-/skin-estuary/horizontal-auto.svg)

fuzzard reviewed (approved):
the android values stuff all looks correct. Thanks for the help.

gade01 commented:
Thanks, that sounds great.

gade01 commented:
If you show the Search button but hide the Favourites button there is an unwanted gap between the buttons.

This can be solved either by turning group control id=10 into a grouplist or by adding a slide animation to the search button and image if the favourites button is hidden.

weblate committed:
Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (85 of 85 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (152 of 152 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Vietnamese (vi_vn))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Portuguese (Portugal) (pt_pt))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 100.0% (3 of 3 strings)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translated using Weblate (English (United States) (en_us))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (English (New Zealand) (en_nz))

Currently translated at 100.0% (3 of 3 strings)

Translated using Weblate (English (Australia) (en_au))

Currently translated at 100.0% (3 of 3 strings)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translated using Weblate (Danish (da_dk))

Currently translated at 100.0% (3 of 3 strings)

Added translation using Weblate (Welsh (United Kingdom) (cy_gb))

Added translation using Weblate (Serbian (latin))

Added translation using Weblate (Persian (Iran) (fa_ir))

Added translation using Weblate (Chinese (Taiwan) (zh_tw))

Added translation using Weblate (Chinese (China) (zh_cn))

Added translation using Weblate (Burmese (my_mm))

Added translation using Weblate (Greenlandic (kl_gl))

Added translation using Weblate (Vietnamese (vi_vn))

Added translation using Weblate (Uzbek (uz_uz))

Added translation using Weblate (Albanian (sq_al))

Added translation using Weblate (Dutch (nl_nl))

Added translation using Weblate (Ukrainian (uk_ua))

Added translation using Weblate (Silesian)

Added translation using Weblate (Turkish (tr_tr))

Added translation using Weblate (Thai (th_th))

Added translation using Weblate (Telugu (India) (te_in))

Added translation using Weblate (Tamil (India) (ta_in))

Added translation using Weblate (Tajik (tg_tj))

Added translation using Weblate (Swedish (sv_se))

Added translation using Weblate (Serbian (sr_rs))

Added translation using Weblate (Persian (Afghanistan) (fa_af))

Added translation using Weblate (Ossetian (os_os))

Added translation using Weblate (Sicilian)

Added translation using Weblate (Maori)

Added translation using Weblate (Slovenian (sl_si))

Added translation using Weblate (Slovak (sk_sk))

Added translation using Weblate (Sinhala (Sri Lanka) (si_lk))

Added translation using Weblate (Russian (ru_ru))

Added translation using Weblate (Romanian (ro_ro))

Added translation using Weblate (Portuguese (Portugal) (pt_pt))

Added translation using Weblate (Portuguese (Brazil) (pt_br))

Added translation using Weblate (Polish (pl_pl))

Added translation using Weblate (Norwegian Bokmål (nb_no))

Added translation using Weblate (Mongolian (mn_mn))

Added translation using Weblate (Maltese (mt_mt))

Added translation using Weblate (Malayalam (India) (ml_in))

Added translation using Weblate (Malay (ms_my))

Added translation using Weblate (Macedonian (mk_mk))

Added translation using Weblate (Lithuanian (lt_lt))

Added translation using Weblate (Latvian (lv_lv))

Added translation using Weblate (Korean (ko_kr))

Added translation using Weblate (Kannada (India) (kn_in))

Added translation using Weblate (Japanese (ja_jp))

Added translation using Weblate (Italian (it_it))

Added translation using Weblate (Armenian (hy_am))

Added translation using Weblate (Indonesian (id_id))

Added translation using Weblate (Icelandic (is_is))

Added translation using Weblate (Croatian (hr_hr))

Added translation using Weblate (Moldavian (Romania) (ro_md))

Added translation using Weblate (Hungarian (hu_hu))

Added translation using Weblate (Esperanto)

Added translation using Weblate (Hebrew (Israel) (he_il))

Added translation using Weblate (Occitan (France) (oc_fr))

Added translation using Weblate (Greek (el_gr))

Added translation using Weblate (Bulgarian (bg_bg))

Added translation using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Added translation using Weblate (Belarusian (be_by))

Added translation using Weblate (Basque (Spain) (eu_es))

Added translation using Weblate (Azerbaijani (az_az))

Added translation using Weblate (Spanish (Mexico) (es_mx))

Added translation using Weblate (Asturian (Spain) (ast_es))

Added translation using Weblate (Arabic (Saudi Arabia) (ar_sa))

Added translation using Weblate (Spanish (Argentina) (es_ar))

Added translation using Weblate (Spanish (Spain) (es_es))

Added translation using Weblate (Hindi (India) (hi_in))

Added translation using Weblate (German (de_de))

Added translation using Weblate (Galician (Spain) (gl_es))

Added translation using Weblate (French (Canada) (fr_ca))

Added translation using Weblate (French (France) (fr_fr))

Added translation using Weblate (Finnish (fi_fi))

Added translation using Weblate (Czech (cs_cz))

Added translation using Weblate (Catalan (Spain) (ca_es))

Added translation using Weblate (Faroese (fo_fo))

Added translation using Weblate (Afrikaans (South Africa) (af_za))

Added translation using Weblate (Estonian (et_ee))

Added translation using Weblate (English (United States) (en_us))

Added translation using Weblate (Amharic (Ethiopia) (am_et))

Added translation using Weblate (English (New Zealand) (en_nz))

Added translation using Weblate (English (Australia) (en_au))

Added translation using Weblate (Danish (da_dk))

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 100.0% (85 of 85 strings)

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kai Sommerfeld <kai.sommerfeld@gmx.com>
Co-authored-by: Marek Adamski <fevbew@wp.pl>
Co-authored-by: Minho Park <parkmino@gmail.com>
Co-authored-by: vdkbsd <valexgus@gmail.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estouchy/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-skins/skin-estuary/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-aac/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/audioencoder-kodi-builtin-wma/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/en_au/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/en_nz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/en_us/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/es_mx/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/hu_hu/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/nl_nl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/pt_pt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/tr_tr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/vi_vn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/kodi-main-android-strings/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-core/screensaver-xbmc-builtin-dim/ko_kr/
Translation: Kodi add-ons: skins/skin.estouchy
Translation: Kodi add-ons: skins/skin.estuary
Translation: Kodi core/audioencoder.kodi.builtin.aac
Translation: Kodi core/audioencoder.kodi.builtin.wma
Translation: Kodi core/kodi main (Android strings)
Translation: Kodi core/screensaver.xbmc.builtin.dim
